gui/default/syncthing/core/syncthingController.js

```diff
@@ -1769,7 +1769,7 @@ angular.module('syncthing.core')
             }
 
             return false
```

Comment:
Shouldn't there be a semicolon here too?

Reply:
Yes, I added the semicolon.
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return, "The `return` statement is affected by automatic semicolon insertion (ASI)."